.drone.star

```diff
@@ -14,65 +14,89 @@ windowsglobalimage="cppalliance/dronevs2019"
 
 def main(ctx):
   return [
+  # Priorities: (no 2a, no betas)
+  #
+  # coverage
+  # latest gcc: 11,14,17,20
+  # latest clang: 11,14,17,20
+  # oldest gcc: 4.8
+  # oldest clang: 3.8
+  # valgrind
+  # asan
+  # tsan
+  # ubsan
+  # docs
```

Comment:
Edit this list to reflect that we put the docs last

Reply:
We didn't. docs is right after valgrind.